UiKitDz/UiKitDz/TabBar/TabBarViewController.swift

```diff
@@ -35,10 +35,11 @@ final class TabBarViewController: UITabBarController {
             image: .chat,
             tag: 1
         )
-        profileViewController.tabBarItem = UITabBarItem(title: Constants.titleProfile, image: .profile, tag: 2)
+        let profileNavViewController = UINavigationController(rootViewController: profileViewController)
+        profileNavViewController.tabBarItem = UITabBarItem(title: Constants.titleProfile, image: .profile, tag: 2)
 
         setViewControllers(
-            [ribbonNavViewController, notificationsNavViewController, profileViewController],
+            [ribbonNavViewController, notificationsNavViewController, profileNavViewController],
```

Comment:
```suggestion
            [feedNavViewController, notificationsNavViewController, profileNavViewController],
```
ribbon обычно используется как лента из какого-то физического материала. В значении ленты соцсетей это Feed. 
Англоязычный пользователь не поймет, у них в языке это никак не связанные слова :)